pkg/builtin/manifest.yml

```diff
@@ -100,6 +100,66 @@ extensions:
               title: Maximum zoom level
               min: 0
               max: 30
+        - id: reTheme
+          title: Publish Theme
+          description: Set your theme.
+          fields:
+            - id: reThemeMode
+              type: string
+              title: Mode
+              defaultValue: preMadeTheme
+              description: Select the theme mode.
+              choices:
+                - key: preMadeTheme
+                  label: Pre-made theme
+                - key: customTheme
+                  label: Custom theme
+            - id: preMadeThemeId
+              type: string
+              title: Pre-made theme
+              description: Select a theme.
+              defaultValue: reEarthDark
+              choices:
+                - key: reEarthDark
+                  label: Re:Earth Dark
+                - key: reEarthLight
+                  label: Re:Earth Light
+                - key: forest
+                  label: Forest
+              availableIf:
+                field: reThemeMode
+                type: string
+                value: preMadeTheme
```

Comment:
```suggestion
            - id: themeType
              type: string
              title: Theme
              description: Select a theme.
              defaultValue: dark
              choices:
                - key: dark
                  label: Re:Earth Dark
                - key: light
                  label: Re:Earth Light
                - key: forest
                  label: Forest
                - key: custom
                  label: Custom
```

These fields can be merged into one. Also, simple IDs is better.

Reply:
I've simplified the IDs, Put those inputs are made separated according to design [here](https://www.figma.com/file/bdnfDaGXGoNfUws4y7JItl/Re-Earth-UI-Gamma?node-id=5899%3A193948)

@rot1024

Reply:
I will discuss about it with @lavalse

Reply:
@yk-eukarya Figma was updated:

![image](https://user-images.githubusercontent.com/3888696/130590549-c989f994-bd73-4fee-b783-12efcfbe1062.png)